Makefile

```diff
@@ -1,5 +1,5 @@
-IMAGE_NAME ?= localstack/localstack
-IMAGE_NAME_BASE ?= localstack/java-maven-node-python
+IMAGE_NAME ?= blafrisch/localstack
```

Comment:
prefer our quay.io repos or if this is meant to be truly public, create an agari/ repo on docker hub.